pythonDiff/differUI.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask, render_template
+
+app = Flask(__name__)
+
+@app.route('/')
+# def hello():
+#     return 'Hello, World!'
+def index():
+    return render_template('index.html')
```

Comment:
I don't see index.html in this change set, I'm guessing that was an accidental exclusion?